example/src/main/java/com/klarna/example/MainActivity.java

```diff
@@ -132,6 +133,7 @@ public void onBuyPressed(View view) {
         } else {
             // Open Klarna registration
             Intent intent = new Intent(this, RegistrationActivity.class);
+            intent.putExtra(RegistrationActivity.EXTRA_SETTINGS, new RegistrationSettingsBuilder().setConfirmedUserDataId("abcd").build());
```

Comment:
Putting Extra in intent is pretty simple/novice/intermediate developer feature... 
But i second explaining the key value assignment here.